Showcase-iOS/Dashboard/Interactor/HomePresenterInteractorImplementation.swift

```diff
@@ -21,8 +21,9 @@ class HomeInteractor: HomePresenterInteractable {
                 self.homePresenter?.onFetchShowcaseAppsFailure(with: .childNotFound)
             } else {
                 let result = snapshot as? DataSnapshot
-                result?.children.forEach { child in
-                    showcaseApps.append(ShowcaseApp(with: child as? [String: Any]))
+                if result?.value != nil {
```

Comment:
update unit tests please

Reply:
Remember we are struggling to abstract the dataSnapshot. Once we have successfully abstracted Datasnapshot then we can update the tests. @edward-mtsh will create a task on Github so that we can remember. Also @VaalSquared says he will give it a go over the weekend. Agree?

Reply:
no tests updated, no merge =) you peeps need to figure it out, if you need help, I dont mind sitting with you, but lets sort this out.